src/components/screens/ScoreSetView.vue

```diff
@@ -303,22 +309,23 @@ export default {
       ensureScoresDataLoaded: scoresRemoteData.ensureDataLoaded
     }
   },
-
   props: {
     itemId: {
       type: String,
       required: true
-    }
+    },
+    GALAXY_URL: String,
+    tool_id: String,
+    sendToGalaxy: String
```

Comment:
any reason for the mix of ALL_CAPS, snake_case, and camelCase here?

Reply:
let's avoid having sendToGalaxy as a prop and sendGalaxy as a function.

function should probably be called sendToGalaxy, name of prop is up to you